### PR TITLE
Add NaverWhale v2.8.105.22

### PR DESCRIPTION
**Pull request by Doosies**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/naver-whale.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+cask "naver-whale" do
+  version "2.8.105.22"
+  sha256 "7c68a5c58f8852db829f1ffdc5a2f592c36a818630b15e78fbcc42464d79615a"
+
+  # https://whale.naver.com/ko/download was verified as official when first introduced to the cask
+  url "http://update.whale.naver.net/downloads/installers/NaverWhale.dmg"
+  appcast "https://whale.naver.com/changelog/"
+  name "네이버 웨일"
```
Comment (Doosies) [inline Casks/naver-whale.rb:8]: desc "Web browser made by Naver Co., Ltd."

**Comment by Doosies** [comment]
how can i fix it?:
C:  5:  3: http://update.whale.naver.net/downloads/ does not match update.whale.naver.net/downloads/installers/NaverWhale.dmg

**Comment by miccal** [comment]
> how can i fix it?:
> C: 5: 3: http://update.whale.naver.net/downloads/ does not match update.whale.naver.net/downloads/installers/NaverWhale.dmg

Just remove the `http://` from the comment above the `url`:
```
# update.whale.naver.net/downloads/ was verified as official when first introduced to the cask
```

File: Casks/naver-whale.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+cask "naver-whale" do
+  version "2.8.105.22"
+  sha256 "7c68a5c58f8852db829f1ffdc5a2f592c36a818630b15e78fbcc42464d79615a"
+
+  # http://update.whale.naver.net/downloads/ was verified as official when first introduced to the cask
```
Comment (miccal) [inline Casks/naver-whale.rb:5]: Please change this to:
```
# update.whale.naver.net/downloads/installers/ was verified as official when first introduced to the cask
```

File: Casks/naver-whale.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  url "http://update.whale.naver.net/downloads/installers/NaverWhale.dmg"
+  appcast "https://whale.naver.com/changelog/"
+  name "네이버 웨일"
```
Comment (miccal) [inline Casks/naver-whale.rb:8]: Please also add the English name:
```
name "NAVER Whale"
```

File: Casks/naver-whale.rb [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  desc "Web browser made by Naver Co, Ltd"
+  homepage "https://whale.naver.com/ko/"
```
Comment (miccal) [inline Casks/naver-whale.rb:10]: Please change this to:
```
homepage "https://whale.naver.com/"
```

File: Casks/naver-whale.rb [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+  app "네이버 웨일.app"
```
Comment (miccal) [inline Casks/naver-whale.rb:12]: Please change this to:
```
app "NAVER Whale.app"
```

Comment (Doosies) [reply Casks/naver-whale.rb]: Fixed everything. Now it works perfectly. Thank you.

**Comment by Doosies** [comment]
Everything works perfectly.

**Review by vitorgalvao — APPROVED** [review]

**Pull request merged**